docs/src/rsh/compute/index.md

```diff
@@ -1831,6 +1831,11 @@ lastConsensusTime()
 The @{defn("lastConsensusTime")} primitive returns the network time of the last publication of the DApp.
 This may not be available if there was no such previous publication, such as at the beginning of an application before the first publication.
 
+```reach
+load: /examples/realtime/index.rsh
+range: 17 - 17
```

Comment:
This example does not add new information as compared to the syntactical reference of `lastConsensusTime`. Samples should provide an example of how the code may be used in a real DApp. 
The idea is that a Reach developer can copy and paste our sample and then modify per the needs of their program. 
In this example, we should show lines 12 - 18.